### DISCUSSION TITLE
如果现在开始学，是否建议直接从InsisVim?

### DISCUSSION THREAD

**Discussion by jingc769** [nshen/learn-neovim-lua#171] [General]
看到这个InsisVim相比于这个repo有了很大改进，自己购买的教程是基于旧的repo的，那我是不是直接从InsisVim开始学习就可以了？

**Comment by nshen** [comment]
由于这个 repo 越改越多，偏离小册太多，所以新建了InsisVim 项目

InsisVim是我常用的，一直在更新，应该可以直接用。

也可以参考小册走一遍流程，到 InsisVim 找到对应插件的最新配置然后拷贝过来，自己尝试一遍

**Reply by jingc769** [reply]
谢谢！
